code/modules/jobs/job_types/gimmick.dm

```diff
@@ -8,6 +8,9 @@
 	supervisors = "no one"
 	selection_color = "#dddddd"
 	chat_color = "#FFFFFF"
+	minimal_player_age = 1
+	exp_requirements = 120
+	exp_type = EXP_TYPE_CREW
```

Comment:
what?

Reply:
Someone in the initial discussion with the present admins requested it. I'm not sure why and I don't even remember who it was tbh.

Reply:
Removed access requirement